src/vector/v.surf.icw/v.surf.icw.py

```diff
@@ -135,14 +112,15 @@
 def cleanup():
     grass.verbose(_("Cleanup.."))
     tmp_base = "tmp_icw_" + str(os.getpid()) + "_"
-    grass.run_command(
-        "g.remove", flags="f", type="raster", pattern=tmp_base + "*", quiet=True
-    )
-    grass.try_remove(TMP_FILE)
+    # grass.run_command('g.list', type='raster', mapset='.', flags='p')
+    result = grass.list_strings("raster", pattern=tmp_base + "*", mapset=".")
+    if len(result) > 0:
+        grass.run_command(
+            "g.remove", flags="f", type="raster", pattern=tmp_base + "*", quiet=True
+        )
```

Comment:
Why the direct call to g.remove does not work without checking the existence first? Too verbose? There is `superquiet=True` if needed.